docs/source/index.rst

```diff
@@ -0,0 +1,49 @@
+======================
+Autometa documentation
+======================
+
+**Autometa: automated extraction of microbial genomes from individual shotgun metagenomes**
+
+An automated binning pipeline for single metagenomes, in particular
+host-associated and highly complex ones. Autometa is copyright 2020
+Ian J. Miller, Evan R. Rees, Izaak Miller and Jason C. Kwan, and is
+released under the GNU Affero General Public License v3 (see :ref:`License`).
+If you find Autometa useful to your work, please cite our Autometa_ paper.
+
+
+Quickstart
+==========
+
+Run with below code::
+
+    python autometa.py metagenome.config
+
+Guide
+=====
+
+.. toctree::
+   :maxdepth: 1
+
+   install
+   databases
+   running-autometa
+   examining-results
+   Scripts/scripts
+   contribution-guidelines
+   api
+   license
+
+Contact
+-------
+For suggestions, queries or appreciation feel free to contact `Dr.JasonKwan <https://apps.pharmacy.wisc.edu/sopdir/jason_kwan/>`_
+at jason.kwan@wisc.edu
+
+
+Indices and tables
+------------------
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
+
+.. _Autometa: https://academic.oup.com/nar/article/47/10/e57/5369936
```

Comment:
This should probably be more prominent, under a section that says something like "Citing Autometa". In the old documentation, I put something like "If you find Autometa useful in your work, please cite our NAR paper [link]". Also, it might be good to have this near the top.

Reply:
![Screenshot from 2020-04-02 14-15-32](https://user-images.githubusercontent.com/43309827/78292610-ec131980-74ec-11ea-8f35-266df5dbc6fa.png)

Right now it appears just above **Quickstart** do you want a separate section for this?
`.. _Autometa: https://academic.oup.com/nar/article/47/10/e57/5369936` just link the Autometa keyword to the url of the paper.

Reply:
The placement is good, but can you please include the full citation for the paper?